CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ endforeach()
 file(GLOB_RECURSE SCHEMAS LIST_DIRECTORIES FALSE RELATIVE "${CMAKE_SOURCE_DIR}/templates" "${CMAKE_SOURCE_DIR}/templates/*")
 foreach(ITEM IN ITEMS ${SCHEMAS})
   if(NOT ITEM MATCHES "^(include/|.*/\\.[^/])" )
-    string(REGEX MATCH "/([-a-zA-Z0-9]+\\.[1-9][0-9]*\\.schema\\.json|[-a-zA-Z0-9_]+\\.[1-9][0-9]*\\.parquetmr\\.txt)$" MATCH ${ITEM})
+    string(REGEX MATCH "/([-a-zA-Z0-9]+\\.[1-9][0-9]*\\.schema\\.json|[-a-zA-Z0-9_]+\\.[1-9][0-9]*\\.parquetmr\\.txt|README.*)$" MATCH ${ITEM})
```

Comment:
@trink Do you see any potential consequences of allowing a README.md file to live inside a `templates` subdirectory and propagate to `schemas`?

Reply:
On my end, this will be fine; anything in the `schemas/metadata` will not be deployed to the GCP MPS branch. We can special-case these files if we do want to include them there in the future.

Reply:
Do you want the readme in the artifact output?  You, would have to check the down stream consumers to make sure they can handle it (I don't believe it would be an issue).  Otherwise, you could remove the else clause and only the schemas will be output.

Reply:
Added a README-specific `elseif` clause to specifically ignore READMEs without dropping other files in the future.